docs/changelog/86110.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 86110
+summary: Add LimitedOffsetsEnum to Limited offset token
+area: Search/Highlighting
```

Comment:
Can you change this to just `Search`? Apparently `Search/Highlighting` doesn't work here, 🤷

Reply:
OK